release-notes.txt

```diff
@@ -1,3 +1,521 @@
+    * boards: Split off 128KiB version of bl*pill
+    + boards/doc: Add a guide to find physical board pins
+    + boards: Initial support for board HiFive1B
+    - boards/mips-malta: remove board.
+    + boards/nrf52: introduce shared dependencies in common/nrf52 and update
+      boards
+    * boards/nrf52832-mdk: enable I2C and use default configuration
+    * board/nucleo-l031k6: Add i2c to periph_conf
+    * board/nucleo-l432kc: Implementing I2C for board
+    * boards: move CPU/CPU_MODEL definition to Makefile.features
+    + boards/p-l496g-cell02: add support
+    - boards: Remove support for the Jiminy-Mega256RFR2
+    * boards/sam0: refactor ADC_0 into plain ADC
+    * boards/slwstk6000b-*: split one board for each module
+    * boards/slwstk6000b: move CPU definiton to Makefile.features
+    + boards/stm32f723e-disco: add initial support
+    * boards/stm32: introduce common timer configurations and use them where
+      possible
+    + boards/stm32l0538-disco: add initial support
+    * stm32-common/spi: allow custom pin modes on spi to minimize power
+      consumption
+    * stm32l1/vendor: update vendor files to v2.3.0
+    * arduino/sketches: build sketches as a module
+    * pic32-wifire: add support for flashing with pic32prog
+
+CPU
+---
+    + cpu/esp32: lwIP netdev
+    * cpu/esp*: various improvements
+    * cpu/kinetis: enable HWRNG for k64f
+    - cpu/mips32r2_generic: remove cpu.
+    * cpu: Moved stdio_init() prior to periph_init() for ARM targets
+    * cpu/nrf5x: decelare radio_nrfble feat for all nrfs
+    + cpu/nrf5x: implement wdt peripheral driver
+    + cpu/sam0_common: add hwrng driver
+    * cpu/sam0_common/gpio: don't hard-code number of ports
+    * cpu/sam0: update doc.txt with new MCU families
+    * cpu/saml21: Make Low-Power SRAM available to programs
+    + cpu/stm32_common: add watchdog for stm32
+    * cpu/stm32f(2|4|7): add riotboot requirements
+    * cpu/stm32l1: optimize power consumption
+    * sam0: allow flashing with JLinkExe
+    + sam0: Implement watchdog driver
+    * cpu/samr30: update vendor files using ASF 3.35.1
+    + cpu/efm32: provide periph_uart_mode
+
+Device Drivers
+--------------
+    + drivers: add mtd wrapper for periph_flashpage
+    + drivers/cc110x: Complete rewrite from scratch, and detailed documentation
+    * drivers/include/periph/eeprom: Changed uint8_t* to void* in API
+    * drivers/periph/i2c: Updated i2c_release() to return void
+    + drivers/ph_oem: support for Atlas Scientific pH OEM sensor
+    + drivers: support for NXP PCA9685 I2C 16-channel, 12-bit PWM controller
+    + usbus: Add CDC-ACM (Serial console) function
+    * periph/timer: Fix return codes of timer API
+    + can: add ncv7356 SW transceiver driver
+    + drivers/lis2dh12: add I2C mode
+    + drivers/periph: add doc on power management aspects
+    * drivers/mrf24j40: add pseudomodules for MRF24J40MA/B/C/D/E
+    * riotboot: define if building the bootloader
```

Comment:
Not sure where to put this, but certainly not in Device drivers

Reply:
Last releases it was [mentioned in sys](https://github.com/RIOT-OS/RIOT/blob/master/release-notes.txt#L934-L935). However, we also could provide a new section for it.

Reply:
Fixed in f5d1301. Moved to sys; there was already a riotboot item there.